includes/functions/settings.php

```diff
@@ -187,17 +188,67 @@ function fields_setup() {
 		]
 	);
 
+	// Add Auth settings section.
+	add_settings_section(
+			'sophi_api_auth',
```

Comment:
Appears there's some extra tabs in this file

Reply:
https://github.com/globeandmail/sophi-for-wordpress/pull/333/commits/a6e5c9a29c3a800de1252b568bc107ec63904ea4